include/villas/nodes/delta_share/DeltaSharingClient.h

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Please follow the style of the other source files:

- Include SPDX headers into header
- Use `snake_case` naming of files.